hr_department_sequence/views/hr_department_view.xml

```diff
@@ -35,7 +35,7 @@
             <field name="model">hr.department</field>
             <field name="inherit_id" ref="hr.view_department_tree"/>
             <field name="arch" type="xml">
-                <xpath expr="//field[@name='complete_name']" position="after">
+                <xpath expr="//field[@name='display_name']" position="after">
```

Comment:
Why this change?

Reply:
This change needs more work in hr_department V10.0 the name his name, and in V11 this is complete_name

Reply:
This is field of displayed name of company. 
https://github.com/OCA/OCB/blob/9d371705228bf58ea9a17255d7677666cb2a3479/addons/hr/views/hr_views.xml#L480